config/evss/mock_letters_response.yml.example

```diff
@@ -412,11 +412,11 @@ default:
         has_survivors_pension_award: false
         monthly_award_amount: 123.0
         service_connected_percentage: 2
-      has_adapted_housing: false
-      has_chapter35_eligibility: true
-      has_death_result_of_disability: false
-      has_individual_unemployability_granted: false
-      has_special_monthly_compensation: false
+        has_adapted_housing: false
```

Comment:
isnt it [already there](https://github.com/department-of-veterans-affairs/vets-api/pull/1134/files/cc7ae63c624243f50c0b3b5b2d3e9da35aad7e79#diff-e9aa151cf2c0087753a7a56888b3e270L44)?